packages/docs/docs/crawler/indexing.md

```diff
@@ -16,14 +16,14 @@ Example
 ```
 
 You can use the attribute multiple times on a single page. The crawler will
-extract the text content from all elements with it and combines them to the page
-index. You can use the `findkit inspect <url>` and `findkit crawl test <url>`
-CLI commands to see what the content is actually selected to the index.
+select the text content from all elements with it and combine them to the page
+index. You can use the `findkit inspect <url>` CLI command to view currently indexed content
+and `findkit crawl test <url>` to see what content would selected to index.
```

Comment:
```suggestion
and `findkit crawl test <url>` to see what content would be selected to index.
```

Reply:
I meant to use `index` as a verb here, but it wasn't very clear. Also it was missing the 'be'
Maybe this would be more clear?

```suggestion
and `findkit crawl test <url>` to see what content would selected to index.
and `findkit crawl test <url>` to see what content would be indexed.
```

@esamattis